docroot/profiles/custom/cgov_site/modules/custom/cgov_image/config/install/core.entity_view_mode.paragraph.ncids_list_item_title_desc.yml

```diff
@@ -0,0 +1,9 @@
+langcode: en
```

Comment:
This should go into cgov_list. This is the view mode for the list item paragraphs themselves. This has nothing to do with the images.